packages/server/dataloader/atlassianLoaders.ts

```diff
@@ -115,48 +119,63 @@ export const jiraIssue = (parent: RethinkDataLoader) => {
             .map((estimate) => estimate.jiraFieldId)
             .filter(isNotNull)
 
-          const issueRes = await manager.getIssue(cloudId, issueKey, estimateFieldIds)
-          if (issueRes instanceof Error) {
-            sendToSentry(issueRes, {userId, tags: {cloudId, issueKey, teamId}})
-            return null
+          const gotIssueCb = async (issueRes: any) => {
```

Comment:
see rewrite in comment above, maybe we can move that whole function into this dataloader?